src/XmlUtils.cc

```diff
@@ -58,10 +74,21 @@ tinyxml2::XMLNode *DeepClone(tinyxml2::XMLDocument *_doc,
 
 /////////////////////////////////////////////////
 std::string ElementToString(const tinyxml2::XMLElement *_elem)
+{
```

Comment:
Can you add a test to cover this method?

Reply:
https://github.com/gazebosim/sdformat/pull/1296/commits/ba8af44e2317e95d046cbf9cfad17f09387ae401